deploy/deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+oc apply -f https://raw.githubusercontent.com/openshift/kata-operator/master/deploy/deploy.yaml && \
+oc adm policy add-scc-to-user privileged -z default -n kata-operator-system && \
```

Comment:
`&& \` needs to be removed from the last statement

Reply:
done, thx!